pkgs/by-name/pr/proton-pass/package.nix

```diff
@@ -25,21 +25,26 @@ stdenvNoCC.mkDerivation (finalAttrs: {
 
   installPhase = ''
     runHook preInstall
-    mkdir -p $out
+    mkdir -p $out/share/proton-pass
     cp -r usr/share/ $out/
-    cp -r usr/lib/proton-pass/resources/app.asar $out/share/
+    cp -r usr/lib/proton-pass/resources/{assets,app.asar} $out/share/proton-pass/
     runHook postInstall
   '';
 
   preFixup = ''
     makeWrapper ${lib.getExe electron} $out/bin/proton-pass \
-      --add-flags $out/share/app.asar \
+      --add-flags $out/share/proton-pass/app.asar \
       --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
       --set-default ELECTRON_FORCE_IS_PACKAGED 1 \
       --set-default ELECTRON_IS_DEV 0 \
       --inherit-argv0
   '';
 
+  postFixup = ''
+    substituteInPlace $out/share/applications/proton-pass.desktop \
+      --replace-fail "Exec=proton-pass %U" "Exec=$out/bin/proton-pass %U"
```

Comment:
Actually we don't want to do this because some desktops cache those files and then start the old version. Also there was recently a tree wide PR removing many of those.

Reply:
Oh, I wasn't aware of that. Thanks for the info! Will get it fixed right away.